### DISCUSSION TITLE
PyCharm - No completions until disabling IdeaVim

### DISCUSSION THREAD

**Discussion by giladbarnea** [community/community#10436] [Copilot Conversations] [answered]
Hi,

After a few weeks of usage, completions stopped showing up entirely. This happened a few weeks ago. I thought it was due to a bad version, but it persisted after updating to 1.1.6.322 today.

[This thread](https://github.com/github/feedback/discussions/8923) describes the same problem, and fortunately the [solution](https://github.com/github/feedback/discussions/8923#discussioncomment-1842427) also worked for me - disabling the IdeaVim plugin.

Before disabling IdeaVim, I set `#com.github.copilot` in Help > Diagnostic Tools > Debug Log Setting, shut down PyCharm, started a new log file, launched PyCharm, and played around a bit in a way that should have displayed suggestions. Here's the log:

[idea.log](https://github.com/github/feedback/files/7918422/idea.log)

**Comment by jansorg** [comment, marked as answer]
Thanks! The next update will have a fix to keep suggestions enabled even when IdeaVim was turned off using the status bar.
